Microsoft.HealthVault.Fhir.UnitTests/Samples/FhirExercise.json

```diff
@@ -0,0 +1,257 @@
+﻿{
+    "resourceType": "Observation",
+    "text": { "div": "New Swim" },
+    "extension": [
+        {
+            "url": "http://healthvault.com/fhir-extensions/thing-flags",
+            "valueString": "None"
+        },
+        {
+            "url": "http://healthvault.com/fhir-extensions/thing-state",
+            "valueString": "Active"
+        },
+        {
+            "url": "http://healthvault.com/extensions/exercise-detail",
```

Comment:
>extensions [](start = 43, length = 10)

we are calling it fhir-extensions for some and extensions for some other, this looks wrong